docs/docs/tutorial-rag.md

```diff
@@ -293,7 +390,7 @@ On a high-level the steps to create custom Scorer are quite simple:
 
   </TabItem>
   <TabItem value="typescript" label="TypeScript">
-    ```plaintext
+    ```typescript
```

Comment:
I wouldn't change this until there is code here - it will cause unwanted syntax highlighting of keywords.